static/app/components/htmlCode.tsx

```diff
@@ -0,0 +1,40 @@
+import 'prism-sentry/index.css';
+
+import {useEffect, useRef} from 'react';
+import styled from '@emotion/styled';
+import beautify from 'js-beautify';
+import Prism from 'prismjs';
```

Comment:
Is `prismjs` a new dependency?

Reply:
It was available in the project I didn't install it. The highlighting that was being used before was not available on the page when I actually dropped the component in. I have been unable to figure out where even it was loaded at.

Reply:
`prismjs` must be listed in package.json if we're going to import it. Otherwise whatever package brought it in might change, and our code will break and no one will know why.